examples/images/voc_sift_fisher.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+#Set environment variables
+: ${KEYSTONE_MEM:=4g}
+export KEYSTONE_MEM
+
+KEYSTONE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/../..
+: ${HADOOP_EXAMPLE_DATA_DIR:=/example_data}
+
+
+#Get the data and copy to HDFS if necessary.
+if [ ! hadoop fs -test -e $HADOOP_EXAMPLE_DATA_DIR/VOCtrainval_06-Nov-2007.tar ]; then
+    wget -O $TMPDIR/VOCtrainval_06-Nov-2007.tar https://s3-us-west-2.amazonaws.com/voc-data/VOCtrainval_06-Nov-2007.tar
+    hadoop fs -copyFromLocal VOCtrainval_06-Nov-2007.tar $HADOOP_EXAMPLE_DATA_DIR
```

Comment:
I have updated the example to not use hadoop and it will use the local filesystem and verified that it runs locally (if you give it enough memory). The way the loader is written it will work fine with data that's on the local filesystem as well as on HDFS, and given the size of the data it won't slow things down dramatically to read off local disk and then shuffle over the network.